components/button/index.css

```diff
@@ -335,5 +339,9 @@ a.spectrum-Button {
 				forced-color-adjust: none;
 			}
 		}
+
+		&.spectrum-Button--negative .spectrum-Button-label {
+			forced-color-adjust: none;
+		}
```

Comment:
Should this be applied to`.spectrum-Button-label` on all the buttons, not just the negative variant? I see it's already applied above to the accent variant, and I am seeing the black text backplate on the primary variant:
![Screenshot 2024-09-13 at 10 33 08 AM](https://github.com/user-attachments/assets/bdac8045-78ec-40c3-81c9-81e85485e76c)

Reply:
Updated it here https://github.com/adobe/spectrum-css/pull/3078/commits/853ddac2d64748b9dadadc5253614ac63b61e0ff#diff-97b5a7e747486aca74b3076354c28a00ef69dab3c91dc0806d926078198571a2R325 ✨